src/transformers/models/whisper/generation_whisper.py

```diff
@@ -231,42 +231,52 @@ def _extract_token_timestamps(
             tensor containing the timestamps in seconds for each predicted token
         """
         # Create a list with `decoder_layers` elements, each a tensor of shape
-        # (batch size, attention_heads, output length, input length).
+        # (batch size * num_beams, attention_heads, output length, input length).
```

Comment:
```suggestion
        # (batch size * num beams, attention_heads, output length, input length).
```
Really big nit, but to keep it consistent (below as well)